libretro/Makefile.common

```diff
@@ -1,3 +1,5 @@
+VERSION     := 0.13.6
```

Comment:
Duplication of the version in the main Makefile, any way to avoid that?

Reply:
I added a new commit below where all 3 Makefile building (Standalone, Libretro, Android) can share one version file. Only way I could think of fixing the problem.